lab_4_fill_words_by_ngrams/main.py

```diff
@@ -273,6 +424,18 @@ def _load_from_json(self) -> dict[tuple[str, int], str]:  # type: ignore
                 or if attribute _json_path has inappropriate extension,
                 or if inappropriate type loaded data.
         """
+        if not (isinstance(self._json_path, str) and self._json_path):
+            raise ValueError
+
+        with open(self._json_path, 'r', encoding='utf-8') as file:
+            q_as = json.load(file)
+
+        if not isinstance(q_as, list):
+            raise ValueError
+
+        self._questions_and_answers = {(q_a['question'], q_a['location']): q_a['answer']
+                                       for q_a in q_as}
+        return self._questions_and_answers
```

Comment:
why both assigning attribute and returning value, i believe it should just be returned